python/lsst/obs/base/yamlCamera.py

```diff
@@ -31,6 +32,7 @@
 __all__ = ["makeCamera"]
 
 
+@lru_cache(maxsize=20)
```

Comment:
Just curious why `maxsize` is altered from the default of 128.  I guess you'll say that we won't have more than 20 cameras anyway, but if that's the argument leaving the default would seem to be OK.  If we did use the default, would holding 128 cameras in memory be an issue?

Reply:
Yes, I was sort of thinking "20 is more cameras than could ever be wanted in a single process" so picked that. I can definitely remove it if it's confusing. Cameras are quite large in general but I can't really foresee a situation where we have 128 instruments in one process and that's our only concern about memory.